deployment/docker-compose.dev.yml

```diff
@@ -6,7 +6,7 @@ services:
     environment:
       - POSTGRES_HOST=relational_db
   relational_db:
-    image: postgres:15.4
+    image: postgres:16.0-alpine
```

Comment:
Could change to 16.1-alpine to match the latest RDS has available ([AWS docs](https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-versions.html)) so our dev and deployment match as closely as possible.

Reply:
Sounds good! Will do

Reply:
Cool thanks. More about version numbers that I just noticed while running tests:

Can we set the postgres image version to 16.1-alpine on:
- `.github/workflows/tests.yaml` (line 20)
- all mentions of `-d postgres` in the `core_backend/Makefile`
- also set `-d qdrant/qdrant` mentions in the `core_backend/Makefile` to v1.5.1 (as per docker compose)

Reply:
Makes sense! Thanks.

Done.